doc/source/reference/frame.rst

```diff
@@ -234,6 +234,8 @@ Reshaping, sorting, transposing
    DataFrame.unstack
    DataFrame.swapaxes
    DataFrame.melt
+   lreshape
```

Comment:
-1 on adding lreshape, wide_to_long is ok

Reply:
Hi @jreback. As requested, I have removed the `lreshape` references.